### PR TITLE
chore(docs): Small changes to docs

### LINKED ISSUE #753: Let the user know there will be a type error sooner in docs section 2
https://nexusjs.org/docs/getting-started/tutorial/chapter-writing-your-first-schema/#your-first-home-grown-query

In this section I got a type error, which kind of threw me into a panic. I had to scroll down a bit to realize this is expected, should the user be warned of this a bit earlier?

### PR DESCRIPTION
**Pull request by MatthewCaseres**
closes #753

Warn user of type error, fix grammar, and make a diff code block for the additional npm scripts because (at least for me) npm init -y includes a "test" script.

### CODE REVIEW

File: docs/content/010-getting-started/03-tutorial/03-chapter-2-writing-your-first-schema.mdx [hunk 1]
```diff
@@ -144,6 +144,8 @@ Your `Post` object is in place now but there's still no way for clients to read
 
 We'll start by letting API clients read the drafts of your blog.
 
+> NOTE: This should throw an error when you save the file. Don't worry, we'll fix it in a second!
```
Comment (jasonkuhrt) [inline docs/content/010-getting-started/03-tutorial/03-chapter-2-writing-your-first-schema.mdx:147]:
```suggestion
> NOTE: This should cause a static type error when you save the file. Don't worry, we'll fix it in a second!
```

**Review by jasonkuhrt — APPROVED** [review]
ty!

**Comment by jasonkuhrt** [comment]
@MatthewCaseres are you ok with my change about the kind of error they should expect to see?

**Comment by MatthewCaseres** [comment]
yeah, change makes sense, if I ever submit a PR take any liberties you want with the proposed changes, your docs!

**Pull request merged**